pkg/services/rpcsrv/server_test.go

```diff
@@ -1468,19 +1607,34 @@ var rpcTestCases = map[string][]rpcTestCase{
 			},
 		},
 		{
-			name:   "unknown contract",
-			params: fmt.Sprintf(`["%s", []]`, util.Uint160{}.String()),
-			fail:   true,
+			name:    "negative, with map in arguments",
```

Comment:
s/`negative, with map in arguments`/`invalid call args`

Reply:
ok